Source/KeyHandler.mm

```diff
@@ -635,7 +649,12 @@ - (BOOL)handleInput:(KeyHandlerInput *)input state:(InputState *)inState stateCa
                 [self _walk];
                 InputStateInputting *inputting = (InputStateInputting *)[self buildInputtingState];
                 stateCallback(inputting);
+                size_t originalCursorIndex = _grid->cursor();
+                if (Preferences.selectPhraseAfterCursorAsCandidate) {
```

Comment:
Shouldn't the predicate be the same as that in lines 542-544? Otherwise this would always move the cursor back by 1 under any circumstances. Also, what happens if `originalCursorIndex` is already 0?

Reply:
I see. Thanks for the clarification!